test/e2e/federated-service.go

```diff
@@ -44,7 +44,7 @@ const (
 	KubeAPIQPS   = 20.0
 	KubeAPIBurst = 30
 
-	FederatedServiceTimeout = 5 * time.Minute
+	FederatedServiceTimeout = 30 * time.Second
```

Comment:
Fair enough.  It seems to take 12 seconds in practise, although I'd like to get that down to close to less than 1 second.

```
Jun 16 17:17:27.632: INFO: Checking that 1 clusters are Ready
Jun 16 17:17:39.763: INFO: Cluster federation-e2e-gce-us-central1-f is Ready
Jun 16 17:17:39.763: INFO: 1 clusters are Ready
```